pkgs/modules/interpreters/python/default.nix

```diff
@@ -0,0 +1,57 @@
+{ pkgs, config, lib, ... }:
+with pkgs.lib;
+let
+  cfg = config.interpreters.python;
+  pythonVersion = cfg.version;
+  python = pkgs.python-versions.${pythonVersion}.python;
+  pypkgs = pkgs.python-versions.${pythonVersion}.pythonPackages;
+  pythonUtils = import ../../../python-utils {
+    inherit pkgs python pypkgs;
+  };
+  pylibs-dir = ".pythonlibs";
+  userbase = "$REPL_HOME/${pylibs-dir}";
+  pythonWrapper = pythonUtils.pythonWrapper;
+  python3-wrapper = pythonWrapper { bin = "${python}/bin/python3"; name = "python3"; aliases = [ "python" "python${pythonVersion}" ]; };
+  sitecustomize = pkgs.callPackage ./sitecustomize { };
+in
+{
+  options = {
+    interpreters.python = {
+      enable = mkModuleEnableOption {
+        name = "Python runtime";
+        description = "Python is a programming language that lets you work quickly and integrate systems more effectively.";
+      };
+
+      version = mkOption {
+        type = types.enum (attrNames pkgs.python-versions);
+        description = "Python version";
+      };
+
+      useSiteCustomize = mkOption {
+        type = types.bool;
+        description = "Whether to use Replit's sitecustomize.py";
+        default = true;
```

Comment:
since we're targeting compat with other systems, it'd be nice to eventually be able to do something like `default = cfg.isReplit`, again just a thought

Reply:
Interesting. I can convert the replit module, and then we'd get another example.